src/lib/mail/flags.ts

```diff
@@ -0,0 +1,28 @@
+import { isUuid } from "../../normalize_id.js";
+import { makeProjectFlagSet } from "../project/flags.js";
+import { assertStatus } from "@mittwald/api-client-commons";
+
+export const {
+  flags: mailAddressFlags,
+  args: mailAddressArgs,
+  withId: withMailAddressId,
+} = makeProjectFlagSet("mailaddress", "m", {
+  normalize: async (apiClient, projectId, id): Promise<string> => {
+    if (isUuid(id)) {
+      return id;
+    }
+
+    const response = await apiClient.mail.listMailAddresses({
+      projectId,
+    });
+    assertStatus(response, 200);
+
+    const address = response.data.find((address) => address.id === id);
```

Comment:
This is unnecessary, since you can just look-up mail addresses by their ID directly. This would only make sense if you would want to use the address string in place of an ID (like with domains, where we allow using the domain name instead of the domain ID).

Reply:
ACK, but shouldn't it be `address.address === id`, then?